mewarpx/mewarpx/plotting.py

```diff
@@ -260,7 +254,8 @@ def slice_array(self):
         self.dim = len(self.array.shape)
         xaxis_idx, yaxis_idx, sliceaxis_idx, self.sliceaxis_str = (
             get_axis_idxs(self.params["xaxis"], self.params["yaxis"],
-                               self.dim))
+                            self.dim)
```

Comment:
Apparently I still fail at spacing in my corrections...
```suggestion
                          self.dim)
```